boot/bootutil/src/swap_move.c

```diff
@@ -140,7 +140,7 @@ swap_read_status_bytes(const struct flash_area *fap,
     found_idx = -1;
     /* skip erased sectors at the end */
     last_rc = 1;
-    write_sz = BOOT_WRITE_SZ(state);
+    write_sz = BOOT_MAX_ALIGN;
```

Comment:
Maybe you should consider making `BOOT_WRITE_SZ` return `BOOT_MAX_ALIGN` for your build?

Reply:
Is my understanding correct on following?

* BOOT_WRITE_SZ()  is a value of state->write_sz;
* state->write_sz is filled in with return value of flash_are_align();
which is, in context of SWAP status table write-operations, the same as BOOT_MAX_ALIGN;

So, is it still required different approaches - macro, API and #define name for the same value?